x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/transforms/TransformHealthChecker.java

```diff
@@ -38,7 +38,8 @@ public enum IssueType {
         PRIVILEGES_CHECK_FAILED("Privileges check failed"),
         TRANSFORM_TASK_FAILED("Transform task state is [failed]"),
         TRANSFORM_INDEXER_FAILED("Transform indexer failed"),
-        TRANSFORM_INTERNAL_STATE_UPDATE_FAILED("Task encountered failures updating internal state");
+        TRANSFORM_INTERNAL_STATE_UPDATE_FAILED("Task encountered failures updating internal state"),
+        TRANSFORM_STARTING("Transform task is automatically retrying its startup process");
```

Comment:
For consistency, could you rename this constant to `TRANSFORM_STARTUP_FAILED` or similar?